### ISSUE TITLE
How to Install in Kodi 15.2 on debian

### ISSUE THREAD

**Issue by rubo77** [jez500/chorus#130]
I installed Kodi as ppacket in debian testing

When I try to add your zip file 0.3.9 I get

    Addon does not have the correct structure

**Comment by GuySa** [comment]
What do you mean by "installed Kodi as ppacket"?

Try installing Chorus via Kodi's repository:
Inside Kodi go to System -> Settings -> Add-ons -> Install from repository -> Web interface -> Chorus.

**Comment by rubo77** [comment]
I refer to the line in your readme:

> Or use the zip in this repo

**Comment by GuySa** [comment]
Yes, I agree it should work, but installing from repository is easier and maintenance-free (as you get updates), so you're recommended to use that method

**Comment by rubo77** [comment]
what this issue is about is that the zip package does not work, so you should either remove the line mentioning that you can install from the package or repair the package

**Comment by GuySa** [comment]
So change the title to something more informative like "Broken zip file" or "Can't install add-on with zip file".
